### PR TITLE
Release 1.10.1

### PR DESCRIPTION
**Pull request by amulya-k**
fix: code change for dynamic option height in vertical layouts.

### CODE REVIEW

**Comment by vinukumar-vs** [comment]
Share the JIRA link in the description of the PR & attach the UI snapshot to the issue.

**Comment by cksachdev** [comment]
Please add a snapshot for any UI issue fixes.

@amulya-k Please do the needful

**Comment by amulya-k** [comment]
@vinukumar-vs   
https://project-sunbird.atlassian.net/browse/SB-7769.
I have added the test scenarios on the above Jira issue.

File: manifest.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "targets": [
         {
             "id": "org.ekstep.questionset",
-            "ver": 1.1
+            "ver": 1.0
```
Comment (vinukumar-vs) [inline manifest.json]: revert manifest.json file changes by merging release-1.10.1 branch into your branch

File: renderer/styles/horizontal_and_vertical.css [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 .mcq-content-container .question-content-container {
     height: 23.33333vh;
-    font-size: 3.9vh;
+    font-size: 14px;
```
Comment (vinukumar-vs) [inline renderer/styles/horizontal_and_vertical.css]: This should be in vh not in px value. It will break in different resolution devices.

**Pull request merged**